train_concept_identification.py

```diff
@@ -13,14 +15,23 @@
 from torch.utils.tensorboard import SummaryWriter
 from yacs.config import CfgNode
 
+from data_pipeline.dummy.dummy_dataset import DummySeq2SeqDataset, build_dummy_vocab
+from data_pipeline.copy_sequence.copy_sequence_dataset import CopySequenceDataset, build_copy_vocab
 from data_pipeline.data_reading import get_paths
 from data_pipeline.vocab import Vocabs
-from data_pipeline.dataset import PAD, EOS, UNK, PAD_IDX
+from data_pipeline.dataset import PAD, BOS, EOS, UNK, PAD_IDX
 from data_pipeline.dataset import AMRDataset
+from data_pipeline.training_entry import ROOT
 from config import get_default_config
 from models import Seq2seq
 from data_pipeline.glove_embeddings import GloVeEmbeddings
 from utils.extended_vocab_utils import construct_extended_vocabulary, numericalize_concepts
+from model.transformer import TransformerSeq2Seq
+from optimizer import NoamOpt
+from yacs.config import CfgNode
+from torchtext.datasets import WikiText2
+from torchtext.data.utils import get_tokenizer
+
 
 FLAGS = flags.FLAGS
```

Comment:
There are quite a lot of flags now. I think we can move the flags and main function to a separate script. But this can be done in a different PR